src/crash_handler.zig

```diff
@@ -737,6 +767,7 @@ fn handleSegfaultPosix(sig: i32, info: *const std.posix.siginfo_t, _: ?*const an
             std.posix.SIG.ILL => .{ .illegal_instruction = addr },
             std.posix.SIG.BUS => .{ .bus_error = addr },
             std.posix.SIG.FPE => .{ .floating_point_error = addr },
+            std.posix.SIG.KILL => .{ .segmentation_fault = addr },
```

Comment:
you cannot intercept SIGKILL. If i am wrong, please add a new crash reason for killed. Killed shouldn't place the text "this is a bug in bun"